std_msgs/Header header
common/Uint8TimeStamped cdcu_bmsmsgperd_err
common/Uint8TimeStamped cdcu_bmsmsgdrop_err
common/Uint8TimeStamped cdcu_bmsmsgchksum_err
common/Uint8TimeStamped cdcu_bmsmsgoffline_err
common/Uint8TimeStamped cdcu_bmslowsoc_alarm
common/Uint8TimeStamped cdcu_bmslowsoc_err
common/Uint8TimeStamped cdcu_bmsovervolt_err
common/Uint8TimeStamped cdcu_bmsundervolt_err
common/Uint8TimeStamped cdcu_bmsovercurt_err
common/Uint8TimeStamped cdcu_bmsovertemp_err
common/Uint8TimeStamped cdcu_bmsunitundervolt_err
common/Uint8TimeStamped cdcu_bmsunitovervolt_err
common/Uint8TimeStamped cdcu_bmsunitovercurt_err
common/Uint8TimeStamped cdcu_bmsunitovertemp_err
common/Uint8TimeStamped cdcu_bmslvbatundervolt_alarm
common/Uint8TimeStamped cdcu_bmslvbatundervolt_err
common/Uint8TimeStamped cdcu_bmslvbatovervolt_alarm